cypress/fixtures/read_from_worpress_file.txt

```diff
@@ -0,0 +1,4 @@
+BP_PHP_VERSION=7.4.x
```

Comment:
Can you try with PHP 8.1 please? because 7.4 is still supported for security patches but it will end in 4 months.
https://www.php.net/supported-versions.php

Reply:
thanks @juadk  for pointing it out.
8.1 did not worked for me but 8.0 did. I can change to that version for now and at least we are covered until November 2023. If we make 8.1 work at some point, we can always update the test.

Reply:
Fine to me :smiley: